core/io/resource_importer.cpp

```diff
@@ -137,6 +137,14 @@ Ref<Resource> ResourceFormatImporter::load(const String &p_path, const String &p
 	}
 
 	Ref<Resource> res = ResourceLoader::_load(pat.path, p_path, pat.type, p_cache_mode, r_error, p_use_sub_threads, r_progress);
+	if (res.is_null()) {
+		// Filename has already been printed in _load()
+		ERR_PRINT("Make sure resources have been imported by opening the project in the editor at least once.");
+#ifdef DEV_ENABLED
+		// Abort to assist CI.
+		CRASH_COND_MSG(!Engine::get_singleton()->is_editor_hint(), "Aborting.");
```

Comment:
I wonder if we should use another `ERR_PRINT()` then exit with status code 1 instead of crashing. Is a crash backtrace useful to have here?